### DISCUSSION TITLE
Can I substitute another HA energy meter for the Octopus Home mini?

### DISCUSSION THREAD

**Discussion by anothermort** [BottlecapDave/HomeAssistant-OctopusEnergy#332] [Q&A] [answered]
Hi 
Love the integration thank you very much.

I do not have a Home mini, but do have a ET112 Victron meter setup in HA.
Is it possible to do the below and sustitute my ET112 meter for the consumed energy sensor for the ET112.
(sensor.victron_grid_l1_energy_forward_30) measured in kWh

Asking as new to HA and tried it but could not get it to work.

Thanks for any help.

Adam

"This is only available if you have an Octopus Home Mini and a smart electricity meter.

    Go to your [energy dashboard configuration](https://my.home-assistant.io/redirect/config_energy/)
    Click Add Consumption under Electricity Grid
    For consumed energy you want sensor.octopus_energy_electricity_{{METER_SERIAL_NUMBER}}_{{MPAN_NUMBER}}_current_consumption
    For Use an entity with current price option you want sensor.octopus_energy_electricity_{{METER_SERIAL_NUMBER}}_{{MPAN_NUMBER}}_current_rate
"

**Comment by BottlecapDave** [comment, marked as answer]
Hi there.

If your meter in HA updates in real time (or near real time), then you can use that sensor as the consumption target and this integration for providing the price. It would be a question that I would direct to the maintainer of the integration that provides your specified meter.

**Reply by anothermort** [reply]
Cheers does seem to work fine. 

Or did.

But Octopus API seems to be a problem at the moment see my other message #337
